src/parser_builder.rs

```diff
@@ -84,6 +100,11 @@ impl ParserBuilder {
         parser.set_threshold(self.threshold);
         parser.set_stop_words(self.n_gazetteer_stop_words.unwrap_or(0),
                               self.additional_stop_words);
+        if let Some(parser_license_info) = self.license_info {
+            let license_info =
+                LicenseInfo::new(parser_license_info.filename, parser_license_info.content);
+            parser.set_license_info(Some(license_info));
```

Comment:
You can directly use `parser.set_license_info(license_info);` if you make the other suggested change on `set_license_info`.